testing/percona-xtrabackup/APKBUILD

```diff
@@ -0,0 +1,64 @@
+# Contributor: Peter Szalatnay <theotherland@gmail.com>
+# Maintainer: Peter Szalatnay <theotherland@gmail.com>
+pkgname=percona-xtrabackup
+pkgver=2.4.6
+pkgrel=0
+pkgdesc='Percona XtraBackup is a MySQL non-blocking backup solution for InnoDB and XtraDB data'
+arch=all
+url='http://www.percona.com/software/percona-xtrabackup/'
+license='GPL'
+makedepends='bash bison cmake ncurses-dev libaio-dev libgcrypt-dev zlib-dev curl-dev libev-dev vim'
+source="https://github.com/percona/percona-xtrabackup/archive/percona-xtrabackup-$pkgver.tar.gz
+        fix-posix_timers.patch
+        "
+
+subpackages="$pkgname-doc $pkgname-test:mytest
+        "
+
+_builddir="$srcdir/percona-xtrabackup-percona-xtrabackup-$pkgver"
+prepare() {
+        local i
+        cd "$_builddir"
+        for i in $source; do
+                case $i in
+                *.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
+                esac
+        done
+}
+
+build() {
+        cd "$_builddir"
+
+        local CFLAGS="-DSIGEV_THREAD_ID=4"
```

Comment:
CFLAGS is a global variable that we set systemwide. To append cflags do:

````sh
CFLAGS="$CFLAGS -DSIGEV_THREAD_ID=4"
````

That way you dont unset the systemwide `-Os` (or what ever default flags is in use)